client/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -8,16 +8,15 @@ import { Button } from '@material-ui/core';
 import Navigation from './Navigation/Navigation';
 import UserDashInfo from './UserDashInfo/UserDashInfo';
 import ScheduleOption from './ScheduleOption/ScheduleOption';
-import { useAuth } from '../../context/useAuthContext';
 import { useSocket } from '../../context/useSocketContext';
 import { useHistory } from 'react-router-dom';
 import { useEffect } from 'react';
 import CircularProgress from '@material-ui/core/CircularProgress';
-import { mockLoggedInUser } from '../../mocks/mockUser';
+import { useAuth } from '../../context/useAuthContext';
 
 export default function Dashboard(): JSX.Element {
   // Test: When authentication or the demo user is complete change the below line to... const { loggedInUser } = useAuth();
-  const loggedInUser = mockLoggedInUser;
+  const loggedInUser = ''; // mockLoggedInUser;
```

Comment:
`mockLoggedInUser` - leaving comments of deleted components may confuse a developer in the future. Please remove it

Reply:
Going to keep the comment since I deleted the old auth context and renamed it to useAuthContext!